make_profiler/__main__.py

```diff
@@ -56,7 +56,7 @@ def main(argv=sys.argv[1:]):
 
     args, unknown_args = parser.parse_known_args(argv)
 
-    in_file = open(args.in_filename, 'r')
+    in_file = check_include_instruction(args.in_filename)
```

Comment:
Does not look good here, "file = check" does not match my expectations (I expect check to return boolean)

Reply:
renamed to **_insert_included_files_**